### PR TITLE
LPS-195244 Massive Preview regeneration: UX improvements

### PR DESCRIPTION
**Pull request by robertoDiaz**
For more info check the screenshots in https://liferay.atlassian.net/browse/LPS-195244

### CODE REVIEW

**Comment by liferay-lima** [comment]
ci:test:sf

File: modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties [hunk 1]
```diff
@@ -14061,6 +14061,7 @@ regenerate-preview-and-thumbnail-of-video-files-in-documents-and-media=Regenerat
 regenerate-preview-and-thumbnail-of-video-files-in-documents-and-media-help=This action does not modify existing video previews.
 regenerate-preview-of-audio-files-in-documents-and-media=Regenerate preview of audio files in documents and media.
 regenerate-preview-of-audio-files-in-documents-and-media-help=This action does not modify existing audio previews.
+regeneration-actions=Regeneration Actions
```
Comment (adolfopa) [inline modules/apps/portal-language/portal-language-lang/src/main/resources/content/Language.properties:14064]: Not sure, but doesn't this sound weird? Maybe "Preview regeneration actions" would be better? I'm fine with sending this to Brian, though; he can probably make a better choice than us.

**Review by adolfopa — APPROVED** [review]

**Comment by adolfopa** [comment]
ci:forward

**Comment by adolfopa** [comment]
ci:forward

**Pull request closed without merging**